x-govuk/components/secondary-navigation/template.njk

```diff
@@ -0,0 +1,14 @@
+<nav class="x-govuk-secondary-navigation {%- if params.classes %} {{ params.classes }}{% endif -%}" aria-label="Sub navigation" {%- for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
+  <ul class="x-govuk-secondary-navigation__list">
+    {% for item in params.items %}
+      <li class="x-govuk-secondary-navigation__list-item {{ "x-govuk-secondary-navigation__list-item--current" if item.current }}{% if item.classes %} {{ item.classes }}{% endif -%}">
+        <a class="govuk-link x-govuk-secondary-navigation__link govuk-link--no-visited-state" href="{{ item.href }}" {{ "aria-current=page" if item.current }}>
+          {{ item.text }}
+          {% if item.current %}
+            <span class="govuk-visually-hidden">Current page</span>
```

Comment:
Is this needed? Wouldn’t the presence of `aria-current` mean this state would be announced twice?

Reply:
Not sure either, but it was already in the code I copied... 😂 

Happy to drop this, unless anyone can remember any good reasons why it was first added...

Reply:
Where did you copy the code from?

Reply:
I can't actually remember when I copied the code from 😬  - I was trying to quick replicate the version we have in production.

Digging through our production git history, it looks like this [was copied](https://github.com/DFE-Digital/early-careers-framework/pull/274/commits/a41842ca4ffef0495724c4cf9b5f3d96ba7a5db6) (2 years ago) from [the version in the HMCTS design system](https://github.com/hmcts/design-system/blob/master/app/components/subnav/_subnav.scss).

Separately, a bunch of other DfE services have a very visually-similar design which uses a different implementation [originally added](https://github.com/DFE-Digital/manage-teacher-training-applications-prototype/pull/196) by @adamsilver (after a crit from you @paulrobertlloyd!).

All of which is good evidence to consolidate all this into a shared component, which can eventually make its way into the GOV.UK Design System 🤞.

Anyway, I've removed the visually-hidden "Current page" text now in 91f2373746c044257f956ba64158ff06746aa429, on the assumption that this is redundant given we’ve added `aria-current="page"` like in the primary nav.